src/components/AccountModule/AccountModule.js

```diff
@@ -189,7 +184,7 @@ function AccountModule() {
                 providerInfo={providerInfo}
                 walletConnectionStatus={walletConnectionStatus}
                 walletListening={walletListening}
-                walletOnline={walletListening}
+                walletOnline={walletOnline}
```

Comment:
Not sure, but I think this should be a fix 😄

Reply:
Whoops! Haha I can _definitely_ tell you this is a fix. 😆